lang/po/Firmware_fr.po

```diff
@@ -2359,8 +2360,8 @@ msgstr "Correct-X"
 
 #. MSG_XFLASH c=18
 #: ../../Firmware/ultralcd.cpp:5462
-msgid "XFLASH init"
-msgstr "Init XFLASH"
+msgid "XFLASH init."
+msgstr "Init. XFLASH"
```

Comment:
@3d-gussner should we consider removing this string from translations? The string is essentially same in all current translations.

Reply:
Actually on a second look, this string is no longer in the firmware 🤔 We need to re-sync the po files.

Reply:
I removed it recently. It was from an unused test menu